.github/workflows/linters.yml

```diff
@@ -0,0 +1,20 @@
+name: Linters
```

Comment:
- [ ] It would be better to have a descriptive README file, this means that the README file must reflect the current project. The project title and description all must correspond to the project